ocsp_status.go

```diff
@@ -38,35 +27,62 @@ func checkFromFile(respFile string) error {
 	return nil
 }
 
+// checkFromCert takes a path to an ASN.1 DER encoded certificate file and constructs and sends an OCSP request
```

Comment:
This line looks a bit long (111 characters). Normally we limit things to 80 characters. Can you move some onto the next line.